### ISSUE TITLE
Help assistance needed....

### ISSUE THREAD

**Issue by Jordaar** [moonstar-x/discord-music-24-7#12]
Is there a possible way for me to make the bot not pause music ?

If yes can you please guide me......thanks!
what should I edit in the code to make this possible.

**Comment by moonstar-x** [comment]
Hey there, the bot is designed to pause when nobody is listening so that the host can save bandwidth.

The easiest change you can make to the code to avoid the pause would be changing the `updateDispatcherStatus()` method in `src/classes/Player.js` to the following:

```js
updateDispatcherStatus() {
    if (!this.dispatcher) {
      return null;
    }
    
    return this.resumeDispatcher();
  }
```

Then again it may have some side effects because the bot was not designed with constant playback.

**Comment by moonstar-x** [comment]
Update:

I'm actually considering having this as a feature that can be selected on the bot configuration. I'm gonna work on this once I'm free, but the change that I showed you should do the trick for now.

**Comment by Jordaar** [comment]
@moonstar-x Thanks!

**Comment by Jordaar** [comment]
And also if it is not pausing so what about the bot's status?

**Comment by moonstar-x** [comment]
I'm sorry, can you rephrase what you tried to say? Do you mean that the change didn't work to make the bot not pause? When you say bot status, do you mean the status of the work on this bot or the message that is displayed under the bot name on the members list of Discord?
